python/sparkdl/graph/spark_utils.py

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
do we want to expose it here? I would prefer to keep it private-ish for the time being, because this should belong to TensorFrames: it calls all sorts of internals and it is not specific to deep learning.